source/extensions/filters/listener/http_inspector/http_inspector.h

```diff
@@ -82,6 +84,10 @@ class Filter : public Network::ListenerFilter, Logger::Loggable<Logger::Id::filt
   Network::ListenerFilterCallbacks* cb_{nullptr};
   Event::FileEventPtr file_event_;
   absl::string_view protocol_;
+  http_parser parser_;
+  static http_parser_settings settings_;
+  uint32_t read_{0};
```

Comment:
if this is only used in HTTP/1.1 path, you may use parser_.nread.

Reply:
Done